src/main/java/org/example/services/JobRoleService.java

```diff
@@ -56,4 +57,13 @@ public JobRoleInfoResponse getJobRoleById(final int id)
     public int getTotalJobRoles() throws SQLException {
         return jobRoleDao.getTotalJobRoles();
     }
+
+    public void checkJobRoleIsValid(final int id)
+            throws SQLException, DoesNotExistException {
+        JobRoleIDResponse jobRoleIDResponse = jobRoleDao.getJobRoleIDByID(id);
```

Comment:
Be consistent with Id across the application.

Reply:
This has been corrected and committed.